lib/RuntimePlugin.js

```diff
@@ -158,6 +159,17 @@ class RuntimePlugin {
 					compilation.addRuntimeModule(chunk, new GlobalRuntimeModule());
 					return true;
 				});
+			compilation.hooks.runtimeRequirementInTree
+				.for(RuntimeGlobals.systemContext)
+				.tap("RuntimePlugin", chunk => {
+					if (compilation.outputOptions.library.type === "system") {
+						compilation.addRuntimeModule(
+							chunk,
+							new SystemContextRuntimeModule()
+						);
+					}
+					return true;
```

Comment:
Do you need to return a bool to keep the loop continuing over each chunk?

Reply:
I don't know - I added this just by following the pattern above (line 160).